base/namedtuple.jl

```diff
@@ -241,6 +241,18 @@ merge(a::NamedTuple{()}, b::NamedTuple) = b
 
 merge(a::NamedTuple, b::Iterators.Pairs{<:Any,<:Any,<:Any,<:NamedTuple}) = merge(a, b.data)
 
+"""
+    merge(a::NamedTuple, b::NamedTuple...)
+
+Perform a recursive merge of two or more named tuples.
```

Comment:
I don't really see this as recursive; maybe we could describe it as left-associative?

Reply:
Oh, it's not actually recursive... Clearly I didn't actually test this properly :smile:
I can change the docstring for this method to indicate that it's left-associate instead; should I also take a stab at making a proper recursive version for this PR (and if so, what should it be named, or should it just be a `recursive::Bool` kwarg to the regular `merge`)?